Makefile

```
@@ -202,3 +202,9 @@ JSONNETFMT_CMD := $(JSONNETFMT) -n 2 --max-blank-lines 2 --string-style s --comm
 .PHONY: jsonnet-fmt
 jsonnet-fmt: | $(JSONNETFMT)
 	PATH=$$PATH:$(BIN_DIR):$(FIRST_GOPATH)/bin echo ${JSONNET_SRC} | xargs -n 1 -- $(JSONNETFMT_CMD) -i
+
+.PHONY: generate-rules-spec
+generate-rules-spec:
```

We might want to rename this as we are not generating a spec, but rather using a spec to generate code
```suggestion
generate-rules-code:
```